3506/3/app/controllers/application_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ApplicationController < Sinatra::Base
+  configure do
+    enable :sessions
+    set :session_secret, 'super secret'
+  end
+
+  not_found do
+    status 404
+    'Something wrong! Try to type URL correctly or call to UFO.'
+  end
+
+  helpers do
+    def current_user
```

Comment:
`current_user` is not a helper it is just a method of ApplicationController. It will be available from views. Also
```ruby
@current_user ||= User.find_by(id: session[:user_id])
```